package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "noobaa-core",
-  "version": "5.3.1",
+  "version": "5.5.0",
```

Comment:
You said that you've updated Azure-sdk version.
I don't see any update for `azure-storage`.
Am I'm missing something?

Reply:
deleted from description, there wasn't an update after all.